test/unit/consultation_test.rb

```diff
@@ -8,7 +8,7 @@ class ConsultationTest < ActiveSupport::TestCase
   should_not_allow_inline_attachments
   should_protect_against_xss_and_content_attacks_on :consultation, :title, :body, :summary, :change_note
 
-  %i[imported deleted].each do |state|
+  %i[deleted].each do |state|
```

Comment:
At first glance, I feel like the 'each' is no longer useful here because the Array only contains one item 🤔 

But I can also see that it's consistent with the approach taken for other states, e.g.
https://github.com/alphagov/whitehall/blob/8392b2f904cd0a5cab875dfc5826377c4b4bb0b8/test/unit/consultation_test.rb#L45

So am inclined to say it's fine to leave it as is, for consistency.